includes/Embed_Block.php

```diff
@@ -53,27 +57,7 @@ public function init() {
 		wp_register_script( 'amp-story-player', 'https://cdn.ampproject.org/amp-story-player-v0.js', [], 'v0', false );
 		wp_register_style( 'amp-story-player', 'https://cdn.ampproject.org/amp-story-player-v0.css', [], 'v0' );
 
-		$asset_file   = WEBSTORIES_PLUGIN_DIR_PATH . 'assets/js/' . self::SCRIPT_HANDLE . '.asset.php';
-		$asset        = is_readable( $asset_file ) ? require $asset_file : [];
-		$dependencies = isset( $asset['dependencies'] ) ? $asset['dependencies'] : [];
-		$version      = isset( $asset['version'] ) ? $asset['version'] : WEBSTORIES_VERSION;
-
-		$dependencies[] = 'amp-story-player';
-
-		wp_register_script(
-			self::SCRIPT_HANDLE,
-			WEBSTORIES_PLUGIN_DIR_URL . 'assets/js/' . self::SCRIPT_HANDLE . '.js',
-			$dependencies,
-			$version,
-			false
-		);
-
-		wp_register_style(
-			self::STYLE_HANDLE,
-			WEBSTORIES_PLUGIN_DIR_URL . 'assets/css/' . self::STYLE_HANDLE . '.css',
-			[ 'amp-story-player' ],
-			$version
-		);
+		$this->load_asset( self::SCRIPT_HANDLE, [ 'amp-story-player' ], self::STYLE_HANDLE, [ 'amp-story-player' ] );
```

Comment:
🚨 This now calls `wp_enqueue_script`. What we want & need here is `wp_register_script`.

There needs to be a `register_asset` method for that.

Reply:
It was using `wp_enqueue_script` before. See https://github.com/google/web-stories-wp/blob/c15743f20af24f945391fde41d3a5aa6e33f1159/includes/Story_Post_Type.php#L285-L291

Reply:
That's in `Story_Post_Type.php`. This here is in `Embed_Block.php`

Reply:
Fixed.